src/NuGet.Services.AzureSearch/Auxiliary2AzureSearch/DownloadSetComparer.cs

```diff
@@ -69,6 +67,16 @@ public class DownloadSetComparer : IDownloadSetComparer
 
             _logger.LogInformation("There are {Count} package IDs with download count changes.", result.Count);
 
+            if (decreaseCount > 0)
```

Comment:
IMO we should log this number regardless of whether or not it's more than zero. If we wanted to graph the number of decreases over time, not having the data points for 0 could lead to an incomplete dataset.

Reply:
Sounds good.